b-Tomas/robot-kinematics
# Forward transformation service

## Changes

Added forward transformation math and its service. Included unit tests.

## How to test

Test the math function with:
```sh
rostest openmanipulator_transformations tests.test
```

To test the service run the usual transformations launch
```sh
roslaunch openmanipulator_transformations transformations.launch
```

and then use the test client to verify that the output for the home pose is the same as the one shown by the controller GUI.
```sh
cd src/robot-transformations/openmanipulator_transformations/scripts/
python3 test_client.py 0 0 0 0  # Response: (0.28600000000000003, -7.83773951454306e-18, 0.20450000000000002)
```

CC @OakNimroW @santiago1311 @JuanM04
